cart-management.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://platinum-ui.netlify.app/components/styles.css" />
+    <link rel="stylesheet" href="cart-management.css">
+    <script src="https://kit.fontawesome.com/228add9a8b.js" crossorigin="anonymous"></script>
```

Comment:
For fontawesome use CDN and import that CDN link in your main CSS file to avoid duplication of this script tag on every html page.